aodn_cloud_optimised/config/dataset/anmn_hourly_timeseries.json

```diff
@@ -1,22 +1,16 @@
 {
   "dataset_name": "anmn_hourly_timeseries",
-  "logger_name": "anmn_hourly_timeseries",
   "cloud_optimised_format": "parquet",
   "metadata_uuid": "7b901002-b1dc-46c3-89f2-b4951cedca48",
-  "gattrs_to_variables": [
-    "site_code"
-  ],
-  "partition_keys": [
-    "site_code",
-    "timestamp",
-    "polygon"
-  ],
+  "logger_name": "anmn_hourly_timeseries",
+  "gattrs_to_variables" : ["site_code"],
+  "partition_keys": ["site_code", "timestamp", "polygon"],
```

Comment:
again, annoying that this changed, not a deal breaker, but makes seeing the actual changes way harder